src/bundled.js

```diff
@@ -99,39 +99,46 @@ export function pluckQuery(... fields) {
   };
 }
 
-function containsField(obj, find) {
-    var args = find.split('.');
-
-  for (var i = 0; i < args.length; i++) {
-    if (!obj || !obj.hasOwnProperty(args[i])) {
-      return false;
+function findFieldOwner(obj, field) {
+  const nestedFields = field.split('.');
```

Comment:
I refactored the code in my new commit to make it looks cleaner and more concise. Plz take a look. (I'll squash it later).